src/execution/vm/llvm_engine.cpp

```diff
@@ -21,6 +21,7 @@
 #include <llvm/Transforms/IPO/AlwaysInliner.h>
 #include <llvm/Transforms/IPO/PassManagerBuilder.h>
 
+#include <iostream>
```

Comment:
Temporary debug include?

Reply:
Good catch! Removed. This reminds me that the other day I was looking at how hard it would be to integrate [include-what-you-use](https://include-what-you-use.org/) into the project - I think it has native CMake support now so it might actually be pretty easy.